inputremapper/gui/components/editor.py

```diff
@@ -294,11 +199,14 @@ def __init__(
         # set the name or combination.beautify as label
         self.label.set_label(self.name)
 
+        self.label.set_margin_top(11)
+        self.label.set_margin_bottom(11)
+
         # button to edit the name of the mapping
         self.edit_btn = Gtk.Button()
         self.edit_btn.set_relief(Gtk.ReliefStyle.NONE)
```

Comment:
```suggestion
```
this relief highlights the button better, although it is still difficult to see. Maybe there is a different theme accent color which we can use to improve the visibility of this button.
![image](https://user-images.githubusercontent.com/31804124/196028566-42792d3e-82f4-4316-af10-5fb4c06fa997.png)

Reply:
I'll keep None since the stock icon is back now